tasks/add-fbc-contribution/tests/test-add-fbc-contribution-pre-ga.yaml

```diff
@@ -0,0 +1,137 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-add-fbc-contribution-pre-ga
+spec:
+  description: Test creating a internal request for the IIB pipeline
```

Comment:
Can you improve the description to describe the specific use case of this test? This applies for the other tests as well.